x-pack/solutions/observability/plugins/apm/common/observability_agent/agent_tool_ids.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const OBSERVABILITY_GET_APM_DOWNSTREAM_DEPENDENCIES_TOOL_ID =
+  'observability.get_apm_downstream_dependencies';
```

Comment:
Is "apm" an important distinction or can we omit it? We should as much as possible avoid the silos between Obs signals and instead think in end results. If dependencies can be derived from logs (questionable but who knows) we should do that as well. 

```suggestion
  'observability.get_downstream_dependencies';
```